meta-python/recipes-devtools/python/python3-lxml_4.5.0.bb

```diff
@@ -1,3 +1,4 @@
 inherit setuptools3
 require python-lxml.inc
 CLEANBROKEN = "1"
+
```

Comment:
this is redundant empty line

Reply:
this empty line was added by the `upgradehelper.py` script, should I delete it anyway?

Reply:
fixed